### ISSUE TITLE
Plotting a network

### ISSUE THREAD

**Issue by sugiYag** [aertslab/scenicplus#495]
hi,
i have scplusmdata.h5mu file and I'm conducting downstream analysis

What I want to know is how to find regulon information in each cell type.

but I couldn't find a tutorial on the recent version.

Can you send any related documents or tutorial files?

**Comment by yojetsharma** [comment]
I think what you're looking for are in API part of the tutorials.

**Comment by sugiYag** [comment]
Which function should I use to get regulon information for each cell type?

**Comment by PhrenoVermouth** [comment]
Exactly, I'm checking through all the outputs, there're pairwise info between TFs, genes and regions, but still cannot figure out the exact content of a signature like Ar_direct_+/+ (What are these genes? What are the enhancers within?). Thanks for answering!

**Comment by SeppeDeWinter** [comment]
Hi @sugiYag and @yojetsharma 

> What I want to know is how to find regulon information in each cell type.

What kind of information are you looking for?

Best,

S
